src/components/RegisterForm/RegisterForm.tsx

```diff
@@ -158,6 +165,14 @@ export const RegisterForm: FC = () => {
     }
   }
 
+  const requiredRule = {required: {value: true, message: 'Toto pole nemože byť prázdne.'}}
```

Comment:
vokan

Reply:
plus podla mna a tohto (https://react-hook-form.com/api/useform/register#options) by to malo byt skor
```suggestion
  const requiredRule = {required: 'Toto pole nemôže byť prázdne.'}
```